code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -276,19 +276,26 @@
 	dragon_fire_line(src, turfs)
 
 //fire line keeps going even if dragon is deleted
-/proc/dragon_fire_line(atom/source, list/turfs)
+/proc/dragon_fire_line(atom/source, list/turfs, frozen = FALSE)
 	var/list/hit_list = list()
 	for(var/turf/T in turfs)
 		if(istype(T, /turf/closed))
 			break
-		new /obj/effect/hotspot(T)
+		var/obj/effect/hotspot/H = new /obj/effect/hotspot(T)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names